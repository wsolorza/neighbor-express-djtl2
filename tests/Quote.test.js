import { shallow } from "enzyme";
import Quote from "../components/Quote.jsx";

let wrapper;

const block = {
  title: 'May the Force be with you',
  body: 'Btn text',
  href: 'www.google.com'
};

beforeEach(() => {
  wrapper = shallow( <Quote block={block} />);
});

describe("Quote block", () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});