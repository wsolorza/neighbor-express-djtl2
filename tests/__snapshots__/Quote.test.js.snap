// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote block renders correctly 1`] = `
<section
  className="usa-section usa-section--dark"
>
  <div
    className="grid-container"
  >
    <h2
      className="font-heading-xl margin-y-0"
    >
      May the Force be with you
    </h2>
    <Button
      href="www.google.com"
      size="big"
    >
      Btn text
    </Button>
  </div>
</section>
`;
